contents.json

```diff
@@ -17554,6 +17554,34 @@
       "date_added": "Mar 29 2019",
       "suggested_by": "@dkhamsing",
       "stars": 287
+    },
+    {
+      "title": "Podcasts",
+      "category-ids": [
+        "audio", 
+        "clone", 
+        "media", 
+        "rxswift"
+      ],
+      "description": "A clone of Apple's Podcasts",
+      "source": "https://github.com/Karambirov/Podcasts",
+      "license": "mit",
+      "tags": [
+        "swift", 
+        "rxswift", 
+        "itunes"
```

Comment:
Can you remove itunes - that’s not a tag we usually track

Reply:
Done